hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupRequest.java

```diff
@@ -163,4 +169,12 @@ public String getYarnPoolName() {
   public void setYarnPoolName(String yarnPoolName) {
     this.yarnPoolName = yarnPoolName;
   }
+
+  private void setContinuousBackupEnabled(boolean continuousBackupEnabled) {
+    this.continuousBackupEnabled = continuousBackupEnabled;
+  }
+
+  public boolean getContinuousBackupEnabled() {
```

Comment:
NIT: if the return type is boolean the method name should be isContinousBackupEnabled